### PR TITLE
fix some config inconsistent

### PR DESCRIPTION
**Pull request by mychaow**
### Content3 ...

<hr>

This PR has:
- [ ] been self-reviewed.
    - [ ] concurrent read
    - [ ] concurrent write
    - [ ] concurrent read and write 
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. 
- [ ] added or updated version, __license__, or notice information
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious 
  for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold 
  for code coverage.
- [ ] added integration tests.
- [ ] been tested in a test IoTDB cluster.

<hr>

### CODE REVIEW

**Review by neuyilan — APPROVED** [review]
LGTM

**Review by wangchao316 — APPROVED** [review]
LGTM

File: cluster/src/main/java/org/apache/iotdb/cluster/config/ClusterConfig.java [hunk 1]
```diff
@@ -59,15 +59,15 @@
 
   private int writeOperationTimeoutMS = (int) TimeUnit.SECONDS.toMillis(30);
 
-  private int catchUpTimeoutMS = (int) TimeUnit.SECONDS.toMillis(60);
+  private int catchUpTimeoutMS = (int) TimeUnit.SECONDS.toMillis(300);
```
Comment (neuyilan) [inline cluster/src/main/java/org/apache/iotdb/cluster/config/ClusterConfig.java:62]: The  `catch_up_timeout_ms` in iotdb-cluster.properties is 60000, should be change together.
`catch_up_timeout_ms=300000`

**Review by neuyilan — CHANGES_REQUESTED** [review]

**Review by neuyilan — APPROVED** [review]
LGTM

**Pull request merged**